docs/source/jit_language_reference_statements.rst

```diff
@@ -0,0 +1,389 @@
+::
+    annotated_assignment_stmt ::=  augtarget ":" expression
+                               ["=" (starred_expression)]
+    `augtarget <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-augtarget>`
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+    `starred_expression <https://docs.python.org/3/reference/expressions.html#grammar-token-starred-expression>`
+
+* `yield expression` is not supported in Torchscript.
+
+The ``raise`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    raise_stmt ::=  "raise" [expression ["from" expression]]
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+
+* Raise statements in TorchScript do not support ``try\finally``.
+
+The ``assert`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    assert_stmt ::=  "assert" expression ["," expression]
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+
+* Assert statements in TorchScript do not support ``try\finally``.
+
+The ``return`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    return_stmt ::=  "return" [expression_list]
+    `expression_list <https://docs.python.org/3/reference/expressions.html#grammar-token-expression-list>`
+
+* Return statements in TorchScript do not support ``try\except\finally``.
+
+The ``del`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    del_stmt ::=  "del" target_list
+    `target_list <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-target-list>`
+
+The ``pass`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    pass_stmt ::= "pass"
+
+The ``print`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    print_stmt ::= "print"
+
+The ``break`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    break_stmt ::= "break"
+
+The ``continue`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    continue_stmt ::= "continue"
+
+The ``import`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    import_stmt     ::=  "import" module ["as" identifier] ("," module ["as" identifier])*
+                     | "from" relative_module "import" identifier ["as" identifier]
+                     ("," identifier ["as" identifier])*
+                     | "from" relative_module "import" "(" identifier ["as" identifier]
+                     ("," identifier ["as" identifier])* [","] ")"
+                     | "from" module "import" "*"
+    module          ::=  (identifier ".")* identifier
+    relative_module ::=  "."* module | "."+
+    `identifier_list <https://docs.python.org/3/reference/lexical_analysis.html#grammar-token-identifier>`
+
+The ``future`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    future_stmt ::=  "from" "__future__" "import" feature ["as" identifier]
+                 ("," feature ["as" identifier])*
+                 | "from" "__future__" "import" "(" feature ["as" identifier]
+                 ("," feature ["as" identifier])* [","] ")"
+    feature     ::=  identifier
```

Comment:
I didn't try but similar to the above, I don't think these are support in TorchScript either.

Reply:
Removed